src/examples/table/TableSort.tsx

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import {
+  Body,
+  Cell,
+  Head,
+  HeaderCell,
+  HeaderRow,
+  Row,
+  SortableCell,
+  Table
+} from '@zendeskgarden/react-tables';
+
+interface IRowData {
+  id: string;
+  subject: string;
+  requester: string;
+  type: string;
+}
+
+type Direction = 'asc' | 'desc' | undefined;
+
+const rowData: IRowData[] = [];
+
+for (let x = 0; x < 10; x++) {
+  rowData.push({
+    id: `row-${x}`,
+    subject: `Custom ticket view ${x + 1}`,
+    requester: x % 2 === 0 ? 'John Smith' : 'Jane Doe',
+    type: x % 3 === 0 ? 'Ticket' : 'Incident'
+  });
+}
+
+const sortData = (tableData: IRowData[], requesterSort: Direction, typeSort: Direction) => {
+  if (!requesterSort && !typeSort) {
+    return tableData;
+  }
+
+  let field: string;
+  let sortValue: Direction;
+
+  if (requesterSort) {
+    field = 'requester';
+    sortValue = requesterSort;
+  } else {
+    field = 'type';
+    sortValue = typeSort;
+  }
+
+  return tableData.sort((a: any, b: any) => {
```

Comment:
These `any` types should be removed because it wipes out all type information within the sort logic. These `any` types can be removed if `field` was typed like this:

```tsx
let field: 'requester' | 'type';
```